extmod/moduplatform.h

```diff
@@ -0,0 +1,105 @@
+/*
+ * This file is part of the MicroPython project, http://micropython.org/
+ *
+ * The MIT License (MIT)
+ *
+ * Copyright (c) 2022 Damien P. George
```

Comment:
Would be better to copy the copyright line from `moduplatform.c`, because the macros below were copied out of that file.

Reply:
Ok.